docs/input-manifest/claudie-custom-ns.md

```
@@ -0,0 +1,66 @@
+    sed -i 's/claudie-webhook$/claudie-webhook-new-namespace/g' claudie.yaml
+    ```
+     2.4. To restrict the namespaces monitored by the Claudie operator (as defined in `claudie.yaml`), add the `CLAUDIE_NAMESPACES` environment variable to the claudie-operator deployment.
+     ```yaml
+     env:
+        - name: CLAUDIE_NAMESPACES
+          value: "new-namespace"
+     ```
+    !!! danger "Updating CLAUDIE_NAMESPACES variable"
+        If there already exist Claudie cluster, make sure to also update the deployment of the existing Claudie operator to reflect the correct namespace.
+
+        If the `CLAUDIE_NAMESPACE` environment variable is not set in the operator, multiple Claudie instances may pick up the same InputManifests, which can lead to the cluster being unintentionally rebuilt. This can result in unexpected behavior and potentially break your Kubernetes cluster.
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)